dlt/extract/extract.py

```diff
@@ -343,6 +343,9 @@ def _extract_single_source(
             "arrow": ArrowExtractor(
                 load_id, self.extract_storage.item_storages["arrow"], schema, collector=collector
             ),
+            "model": TextExtractor(
```

Comment:
this will be `ModelExtractor` and it will contain logic to normalize and rewrite models (ie. to add `_dlt_load_id`. you can already rename that